src/backends/sentry_backend_breakpad.cpp

```diff
@@ -113,8 +113,8 @@ sentry__breakpad_backend_callback(
 
         if (should_handle) {
             sentry_value_t event = sentry_value_new_event();
-            sentry_envelope_t *envelope
-                = sentry__prepare_event(options, event, NULL);
+            sentry_envelope_t *envelope = sentry__prepare_event(
+                options, event, nullptr, !options->on_crash_func);
```

Comment:
without informing `sentry__prepare_event()` whether `before_send()` should be invoked, we cannot make sure that a set `on_crash()` prevents the execution of the `before_send()` callback.